app/front/scripts/services/downloader.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
@akariv  I got problems with API and babble.ui "without proxy". but it works now. so I can disable proxy in `downloader` service.